README.md

```diff
@@ -165,3 +166,10 @@ The files in this folder are IBA probe json payloads that are represented as
 [JINJA templates](http://jinja.pocoo.org/docs/2.10/templates/). You need to use
 aos-cli to load these probes on to AOS server. The command to use in aos-cli is
 `probe create --blueprint <id> --file </path/to/template/file> [<additional_args>]`
+
+
+## Probe graph queries
+
+Authoring new probes requires familiarity with graph queries to select the elements
+of interest whose telemetry is ingested into the probes. Refer to
+[graph queries](graph_queries/README.md) for more details.
```

Comment:
I think we may also need some explanation of the collectors,  or at least a reference to telemetry: https://files.apstra.com/docs/telemetry.html

Reply:
Will do